constuctor_assignment.js

```diff
@@ -0,0 +1,188 @@
+console.log("Question 1")
+
+function Person(firstName, lastName, middleName) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.middleName = middleName;
+}
+
+Person.prototype.fullName = function () {
+    console.log(`${this.firstName} ${this.middleName} ${this.lastName}`)
+}
+let karen = new Person("Karen", "Morisset", "A")
+let brutus = new Person("Kevin", "Brutus", "A")
+
+karen.fullName()
+brutus.fullName()
+
+console.log("Question 2")
+function Book(title, author, rating) {
+    this.title = title;
+    this.author = author;
+    this.rating = rating;
+}
+
+Book.prototype.isGood = function () {
+    if(this.rating >= 7) {
+        return true 
+    } else {
+        return false
+    }
+}
+let book1 = new Book("Notebook", "Karen", 9)
+let book2 = new Book("PenPal", "Brutus", 6)
+
+console.log(book1.isGood())
+
+console.log("Question 3")
+
+function Dog(name, breed, mood, hungry) {
+    this.name = name;
+    this.breed = breed;
+    this.mood = mood;
+    this.hungry = hungry;
+}
+
+Dog.prototype.playFetch = function () {
+    this.mood = "playful", this.hungry = true
+    console.log("Ruff!")
+}
+
+Dog.prototype.feed = function () {
+    if(this.hungry === true){
```

Comment:
Because this.hungry evaluates to a boolean there is no need to do the === true. It's not wrong, just not needed.